### ISSUE TITLE
How to implement this model to larger scale image task?

### ISSUE THREAD

**Issue by hmf21** [leftthomas/CapsNet#4]
Hi @leftthomas 
the part of reconstruction for Capsnet works out by using nn.Linear as the last layer in self.decoder and for MNIST dataset, it is 784 output channel.
However, if we want to implement Capsnet to another image classification dataset such as 256*256 colored images, whether it has too much paremeters to be trained for reconstruction part. What should we do when facing this problem?
Very appreciate!

**Comment by leftthomas** [comment]
@hmf17 CapsNet requires many parameters, and need very powerful GPU to train, and I think it is not suitable for large size image till now.

**Comment by hmf21** [comment]
Hi @leftthomas Thanks for your reply, and according to your explanation CapsNet doesn't fit this task.
I am wondering if we remove the reconstruction part in loss function, can it be implemented in such large size image classification task?

**Comment by leftthomas** [comment]
@hmf17 no, the CapsNet is FC with for loop in essence, so it requires many parameters.
